client/my-sites/checkout/composite-checkout/test/credit-card.tsx

```diff
@@ -66,6 +66,11 @@ function ResetCreditCardStoreFields() {
 	} );
 }
 
+jest.mock( '@automattic/shopping-cart', () => ( {
```

Comment:
It's not a big deal but I wonder if these mocks need to be present any more? In my testing the tests pass without them.